version.js

```diff
@@ -25,20 +68,35 @@ async function createRelease(version, releaseNotes, config) {
 async function getCurrentVersion(config) {
     const data = await config.octokit.git.listMatchingRefs({
         ...github.context.repo,
-        namespace: 'tags/',
+        ref: 'tags/',
```

Comment:
I think this line incorrect previously: https://octokit.github.io/rest.js/v18#git-list-matching-refs. `namespace` isn't a parameter. `ref` is. So I think what it was doing previously was listing all refs, not just tags.

Also, out of scope for this PR, but this function only returns the latest 30 tags. It probably should [paginate](https://octokit.github.io/rest.js/v18#pagination).

> All endpoint methods starting with .list* do not return all results at once but instead return the first 30 items by default, see also GitHub’s REST API pagination documentation.